### DISCUSSION TITLE
Please add Testing Smart Contracts with Ganache: A Comprehensive Guide in the Truffle Suite Documentation

### DISCUSSION THREAD

**Discussion by hardiktheprogrammer** [ConsenSys-archive/truffle#6223] [Ideas]
I would like to add  on  Documentation How to test smart contract on  Ganache

**Comment by MadelineMurray** [comment]
hi @hardiktheprogrammer - Truffle and Ganache are being sunset. For more information and migration options, click [here](https://consensys.io/blog/consensys-announces-the-sunset-of-truffle-and-ganache-and-new-hardhat?utm_source=github&utm_medium=referral&utm_campaign=2023_Sep_truffle-sunset-2023_announcement_).
